bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Ballot.h

```diff
@@ -27,6 +27,12 @@ namespace ElectionGuardInterop
 
             this->_instance = new electionguard::ExtendedData(_value, length);
         }
+        internal : ExtendedData(std::unique_ptr<electionguard::ExtendedData> other)
```

Comment:
all of the c++/cli classes should now have this pattern, where internally in the library they can take ownership of the object or only wrap its access depending on context:

- smart pointer: wrapper takes ownership
- c-style pointer: wrapper does not own
- c-style constant pointer: wrapper does not own